### DISCUSSION TITLE
The reading of Analog 2 is always 0

### DISCUSSION THREAD

**Discussion by shaisab** [pyPhotometry/code#12] [Q&A] [answered]
Hello,

We have just received a new pyPhotometry board.

We are using the system at '2 colors time division' mode. However, Analog 2 appears to not work properly. The reading of Analog 2 is always 0. Resetting the pyBoard did not solve the problem. Installing the new release of pyPhotometry also did not help.

Both LEDs are ok. Both LED drivers are also fine (for testing, I set the sampling frequency to 2Hz, and I can see that each LED is turning on every 1 second).

Any ideas how to resolve this issue?

Thanks.

**Comment by ThomasAkam** [comment, marked as answer]
Hi @shaisab,

Are you using an optical setup with two LEDs and two photorecievers (e.g. for red/green fluorphores) or an optical setup with two LEDs but one photoreciever (e.g. for green + isosbestic)?  If it is the latter then you need to aquire signal in '1 color time division' mode.

If you are using a setup with 2 LEDs and 2 photorecievers, please try the following:

1.  Aquire some signal in '2 channel continuous' mode and let me know what the analog 2 reads.  This is useful for debugging as in continous mode you see the raw photoreciever voltage without baseline subtraction. 
2. Try swapping the LED+photoreciever connected to channel 1 with those connected to channel 2.  If the problem is with the pyPhotometry board then you should now have signal on channel 2 and no signal on channel 1.

let me know how you get on.

Thomas

**Reply by shaisab** [reply]
Thomas, 
Thanks a lot for your quick response.

Our setup comprises two LEDs but one photoreciever. Therefore, we should have used the '1 color time division' mode.
Thank you for making me understand my stupid mistake :-)
